packages/babel-preset-latest/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-preset-latest
+
+> Babel preset including es2015, 2016, 2017.
```

Comment:
@hzoo 

> I think we'll want to explain what "latest" means here.

I agree with you but I am not sure how it is called so I just use latest.  What is the purpose of it ? I think it help people try all features very easy instead of installing a lot of plugins.